pkg/model/proxy.go

```diff
@@ -418,9 +419,23 @@ const (
 	k8sSeparator = "."
 )
 
-// GetLocalityLabel returns the locality from the supplied label. Because Kubernetes
+func GetLocalityLabel(labels map[string]string) string {
```

Comment:
Can you please add a comment explaining the order. For example...

// GetLocalityLabel returns the locality label from the labels map.
// It checks labels in the following priority order
//  1. topology.istio.io/locality
//  2. istio-locality (legacy label for backwards compatibility)
// Returns empty string if no locality label is found.